var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
 
     version('1.1.6', '5f13a0841a61f7fc295c514490d120d0',
             url="https://pypi.python.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz")
 
-    depends_on('python@2.4:2.8,3.3:')
+    depends_on('python@2.4:3.3.99')
```

Comment:
I don't explicitly know. I confirmed once that it builds with 3.2.6, but nothing else. According to its description it should already be included in the standard installation of 3.4+, nevertheless it builds there...

However I don't have access to this system right now and only my home system (bash on windows) and here python 3.2. doesn't build at all... I hope that our ```afs``` is fixed tomorrow, then I can confirm that at work.

A more general question: Does ```:3.3``` include ```3.3.1``` or does it not?